src/app/containers/StoryPromo/index.test.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import { shouldMatchSnapshot } from '../../helpers/tests/testHelpers';
+import { RequestContextProvider } from '../../contexts/RequestContext';
+import deepClone from '../../helpers/json/deepClone';
+import StoryPromo from '.';
+
+let item;
+const completeItem = {
+  headlines: {
+    headline: 'A headline',
+  },
+  summary: 'Summary text',
+  timestamp: 1556795033,
+  indexImage: {
+    path: '/cpsprodpb/0A06/production/image.jpg',
+    height: 1152,
+    width: 2048,
+    altText: 'Image Alt text',
+    copyrightHolder: 'Image provider',
+  },
+};
+
+describe('StoryPromo Container', () => {
+  shouldMatchSnapshot(
```

Comment:
Likewise here, these snapshot tests are redundant - they don't provide extra test coverage, so should be removed.

Reply:
cc https://github.com/bbc/simorgh/pull/1640#discussion_r281770729